### PR TITLE
Enable GlutenParquetRebaseDatetimeV1Suite - gluten not considers rebase mode 

### PR DESCRIPTION
**Pull request by jinchengchenghh**

### CODE REVIEW

**Review by rui-mo — COMMENTED** [review]
Can we rewrite the file path a bit to avoid putting these test data into Gluten? Maybe you could refer to [GlutenCSVSuite](https://github.com/oap-project/gluten/blob/main/gluten-ut/spark32/src/test/scala/org/apache/spark/sql/execution/datasources/csv/GlutenCSVSuite.scala#L29-L30) to read test data based on Spark home path.

**Comment by jinchengchenghh** [comment]
The limitation is described in https://github.com/oap-project/gluten/pull/958/files#diff-40805fca0693503ffa4448b8b6f8ec7256473d5e1dfc3dc2bbbae2ded5cf5012

**Comment by jinchengchenghh** [comment]
> Can we rewrite the file path a bit to avoid putting these test data into Gluten? Maybe you could refer to [GlutenCSVSuite](https://github.com/oap-project/gluten/blob/main/gluten-ut/spark32/src/test/scala/org/apache/spark/sql/execution/datasources/csv/GlutenCSVSuite.scala#L29-L30) to read test data based on Spark home path.

fixed

File: backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/VeloxBackend.scala [hunk 1]
```diff
@@ -42,6 +43,7 @@ object VeloxBackendSettings extends BackendSettings {
         // Unsupported types are prevented.
         fields.map(_.dataType).collect {
           case _: BooleanType =>
+//          case DateType =>
```
Comment (rui-mo) [inline backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/VeloxBackend.scala:46]: How should we handle DateType? Looks a little confusing here.

Comment (jinchengchenghh) [reply backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/VeloxBackend.scala]: I forget to delete this line, deleted.

**Review by rui-mo — APPROVED** [review]

**Pull request merged**